app/src/main/java/com/globant/practice/presentation/presenter/WebClientPresenter.java

```diff
@@ -0,0 +1,81 @@
+package com.globant.practice.presentation.presenter;
+
+import android.os.Bundle;
```

Comment:
A presenter must not know nothing about Android. It should be isolated from the framework